requirements.txt

```diff
@@ -1,11 +1,13 @@
-pip~=22.0
+pip==23.0
 ninja
 cmake
-custatevec-cu11
+custatevec-cu12
```

Comment:
So we are building exclusively with CUDA 12 now, or should this be cu11?

Reply:
Should be cu11. Thanks.